steps/telegram-notify/1.0.1/step.yml

```diff
@@ -0,0 +1,59 @@
+title: Send Telegram Message
+summary: |
+  Notify telegram group or chat on builds status.
+description: |
+  This step will send a message to the provided [telegram](https://core.telegram.org/bots) chat.
+  Includes the build status and download URL. You can also add a custom message to be sent with the notification.
+  This step always runs, no matter if build succeded or failed.
+website: https://github.com/mcaldas/bitrise-step-telegram-notify
+source_code_url: https://github.com/mcaldas/bitrise-step-telegram-notify
+support_url: https://github.com/mcaldas/bitrise-step-telegram-notify/issues
+published_at: 2018-10-25T02:28:04.592933592+03:00
+source:
+  git: https://github.com/mcaldas/bitrise-step-telegram-notify.git
+  commit: 6d6792f5141c662e888f42f5fbd96ebd18480b06
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      The bot token.
+      Can also be taken from env var `TELEGRAM_BOT_TOKEN`
+    is_required: true
+    title: Telegram Bot Token
+  telegram_bot_token: $TELEGRAM_BOT_TOKEN
```

Comment:
Inputs like tokens should be usually sensitive: https://devcenter.bitrise.io/builds/sensitive-input-field/

Reply:
👍 good catch